docker-compose.deps.py2.yml

```diff
@@ -10,16 +10,16 @@
 version: '2.1'
 
 services:
-  pip:
+  hepcrawl_testing_image:
     build:
-      context: ${PWD}/tests
-      dockerfile: Dockerfile.hepcrawl_base
+      context: ./
+      dockerfile: tests/Dockerfile.hepcrawl_base
+      args:
+        - GID=${GID:-1000}
+        - UID=${UID:-1000}
     image: hepcrawl_base
+    user: "${UID:-1000}:${GID:-1000}"
     tty: true
-    environment:
-      - BASE_USER_UID=${BASE_USER_UID:-1000}
-      - BASE_USER_GIT=${BASE_USER_GIT:-1000}
     command: bash -c "pip install -e .[all] && pip freeze"
```

Comment:
I got confused by this, I think you can remove it (the py3 version is simply `"true"`)

Reply:
Ah, right. I missed it.